### ISSUE TITLE
Data issue 

### ISSUE THREAD

**Issue by minhnimble** [SeintApril/Survey#7]
When seeing the Surveys List screen, the data list of surveys are actually hardcoded and not coming from the backend. I am not really sure why do you do this but from the code challenge's requirement, it is required to fetch the surveys list from an API -> `The list of surveys must be fetched when opening the application.`

**Comment by SeintApril** [comment]
Yes. I am trying to fetch the surveys list from an API but it is a little bit hard for me. I took a long time to implement it but I can't be done and leave with hard coded  😢
